### ISSUE TITLE
isin method not founf in dataframe  object

### ISSUE THREAD

**Issue by jabellcu** [dask/dask#2071]
python 3.5, dask 0.14.0

I get the following error when trying to use:

`dask_dataframe[['col1', 'col2']].isin(pandas_Series).all(axis=1)`

```
AttributeError                            Traceback (most recent call last)
<ipython-input-21-64b84af10254> in <module>()
----> 1 II_mask = PaxData_filtered[['Origin Name', 'Destination Name']].isin(KeyStations).all(axis=1)

C:\Users\jabellan\AppData\Local\Continuum\Anaconda3\lib\site-packages\dask\dataframe\core.py in __getattr__(self, key)
   2121             return new_dd_object(merge(self.dask, dsk), name,
   2122                                  meta, self.divisions)
-> 2123         raise AttributeError("'DataFrame' object has no attribute %r" % key)
   2124 
   2125     def __dir__(self):

AttributeError: 'DataFrame' object has no attribute 'isin'
```

However, 'isin' method is implemented according to the documentation:

[http://dask.pydata.org/en/latest/dataframe-api.html](url)

Thanks for your help!

**Comment by jcrist** [comment]
The `isin` method is implemented for `Series`, but not for `DataFrame` (if you look, you'll see that `isin` is in the docs only for `Series`). Implementing it for `DataFrame` wouldn't be too hard, if you'd care to make a pull request.

**Comment by jabellcu** [comment]
Thanks for the clarification. I will give it a try, but it is probably above my skill level.

**Comment by jcrist** [comment]
This was fixed by #2558. Closing.
